tests/migration_test.go

```diff
@@ -1766,11 +1766,14 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 						Expect(err).ToNot(HaveOccurred(), "Should successfully get new VMI")
 						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, tests.NamespaceTestDefault)
 
-						if newvmi.Status.Interfaces[0].IP == vmiPod.Status.PodIP {
-							return nil
+						Expect(newvmi.Status.Interfaces[0].IP).To(Equal(vmiPod.Status.PodIP))
```

Comment:
1. It is incorrect to assert here as it will exit on first fail and never retry.
2. There are multiple tests checks:
    - Both IP and IPs is checked
    - The IPs is checked that it is not empty
    - The number of IP/s is the same.
    -  Individual IP/s are equal.

For the last point, I think we can:
- Check only the IPs, the fact that IP == IPs[0] should be a fact and tested once and not on every test.
- Check len > 1
- For-loop and check value equality.

Reply:
1. i also thought about it, but saw we are doing that all over, please search `Eventually` and see,
we can inject an error and make sure what is the behavior or find an article,
how you suggest to proceed ? lets mitigate this first and then decide ?

Reply:
There are cases where there is no point in trying again and cases where there is. The retry is done to wait for the IP values to get updated.

Reply:
about "Check only the IPs, the fact that IP == IPs[0] should be a fact and tested once and not on every test."
i think we should check it, at least on some tests. its a standalone field, and the logic is important in case it will break.

Reply:
Done (all) , please take a look